common/types/activation_test.go

```diff
@@ -26,12 +29,23 @@ func TestRoundEndSerialization(t *testing.T) {
 }
 
 func TestActivationEncoding(t *testing.T) {
-	types.CheckLayerFirstEncoding(t, func(object types.ActivationTx) types.LayerID { return object.PubLayerID })
+	var object types.ActivationTx
+	f := fuzz.NewWithSeed(1001)
+	f.Fuzz(&object)
+
+	buf := bytes.NewBuffer(nil)
+	enc := scale.NewEncoder(buf)
+	_, err := object.EncodeScale(enc)
+	require.NoError(t, err)
+
+	epoch := types.EpochID(rand.Uint32())
```

Comment:
The assigned value here is overwritten in the next line

```suggestion
	var epoch types.EpochID
```